addon-test-support/scroll-into-view-mock.js

```diff
@@ -0,0 +1,32 @@
+import { find } from '@ember/test-helpers';
+
+export default function mockScrollIntoView() {
+  let elementsInvokedOn = [];
+  let mockScrollIntoViewFunction = function () {
+    elementsInvokedOn.push(this);
```

Comment:
Question: Should we call the real method as well after recording the invocation element, in case the user still wants the behavior to execute for some reason? e.g. `preExistingScrollFunction.call(this, ...arguments)` or whatever.

Reply:
we can, not sure if that might have undesirable side effects in some tests though assume for example that the scroll is still going when you transition from one test to the next.

Reply:
I'm 50/50 either way, so I wouldn't push strongly for this; it was just a thought that occurred to me. I'm curious what @jdkahn @ygongdev think, though!

Reply:
I wouldn't call the actual function, at most I'd do something like `await triggerEvent('scroll', ...);`.
Imo, it is like how you would prefer `await click(button)` over `button.click()` in a test?

Reply:
I think that would be even more confusing. The user has already triggered the method on purpose. We should either call it or not, but I don't know why we would call `triggerEvent` here.

Reply:
Hmm, let me rephrase. 

We might not want to call the native `scrollIntoView` because the scroll is `non-deterministic` and it might not handle the `ember-testing` container well, which is #2.

I mention `await triggerEvent(..., 'scroll')` because it seems to mimics #2 deterministically during testing. Looks like there is also a new [`scrollTo`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#scrollto) API as well.

This also doesn't have to live internally and we can leave it up to the consumer.

Reply:
Ah, I see. It might be worth adding the `scrollTo` call if we can ensure it's called in a way that would mimic scrollIntoView.

Reply:
I disagree with causing DOM changes in a test by default, I think it's unneeded possible source of errors. We can provide the ability for the caller to pass in a parameter or something to invoke the native method as well.

Reply:
It is up to you, but if you want to provide a helper for the caller, you can just expose another function right?
```
this.mockHelperFunctions.invokeScrollIntoView(element);
```

Reply:
I'm fine not adding this behavior until we have a use-case for it.

Reply:
@elwayman02 that would be my vote as well.

Reply:
I'm fine with this, adding it later wouldn't be a breaking change anyways.